district/src/main/java/org/government/authentication/authenticator/SimpleAuthenticator.java

```diff
@@ -0,0 +1,20 @@
+package org.government.authentication.authenticator;
+
+import org.government.api.User;
+
+import com.google.common.base.Optional;
+
+import io.dropwizard.auth.AuthenticationException;
+import io.dropwizard.auth.Authenticator;
+import io.dropwizard.auth.basic.BasicCredentials;
+
+public class SimpleAuthenticator implements Authenticator<BasicCredentials, User> {
+
+	public Optional<User> authenticate(BasicCredentials credentials) throws AuthenticationException {
+		if ("secret".equals(credentials.getPassword())) {
+			return Optional.of(new User(credentials.getUsername(), "sample", "sample"));
```

Comment:
Open an issue to pull records from Database and validate against it for user password authentication and write TODO comment.